### ISSUE TITLE
拉到本地后，运行yarn serve 报错。

### ISSUE THREAD

**Issue by high9527** [anncwb/vben-admin-thin-next#8]
(node:28344) UnhandledPromiseRejectionWarning: Error: [vite]: Rollup failed to resolve import "@vue/composition-api" from "node_modules\vue-demi\lib\v2\index.esm.js".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`
    at onRollupWarning (E:\dengxj\vue\vben-test-001\node_modules\vite\dist\node\chunks\dep-6f99a5ef.js:40308:19)
    at Object.onwarn (E:\dengxj\vue\vben-test-001\node_modules\vite\dist\node\chunks\dep-6f99a5ef.js:71042:17)
    at Object.onwarn (E:\dengxj\vue\vben-test-001\node_modules\vite\node_modules\rollup\dist\shared\rollup.js:19541:20)
    at ModuleLoader.handleResolveId (E:\dengxj\vue\vben-test-001\node_modules\vite\node_modules\rollup\dist\shared\rollup.js:18451:26)
    at E:\dengxj\vue\vben-test-001\node_modules\vite\node_modules\rollup\dist\shared\rollup.js:18418:22
    at async Promise.all (index 1)
    at async ModuleLoader.fetchStaticDependencies (E:\dengxj\vue\vben-test-001\node_modules\vite\node_modules\rollup\dist\shared\rollup.js:18416:34)
    at async Promise.all (index 0)
    at async ModuleLoader.fetchModule (E:\dengxj\vue\vben-test-001\node_modules\vite\node_modules\rollup\dist\shared\rollup.js:18393:9)
    at async Promise.all (index 0)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:28344) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:28344) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by liuwave** [comment]
使用`yarn reinstall` 重装，这个错误会消失，但会出现新的错误：

```
19:34:54 [vite] Internal server error: "version" is a required argument.
      at Object.getArg (C:\..\vben-admin-thin-next\node_modules\vite\dist\node\chunks\dep-6f99a5ef.js:25004:11)
      at new BasicSourceMapConsumer (C:\work\chengdu-othter\kjc\vben-admin-thin-next\node_modules\vite\dist\node\chunks\dep-6f99a5ef.js:26642:24)      at new SourceMapConsumer (C:\..\vben-admin-thin-next\node_modules\vite\dist\node\chunks\dep-6f99a5ef.js:26370:7)      
      at merge (C:\..\vben-admin-thin-next\node_modules\vite\dist\node\chunks\dep-6f99a5ef.js:27954:24)
      at TransformContext._getCombinedSourcemap (C:\..\vben-admin-thin-next\node_modules\vite\dist\node\chunks\dep-6f99a5ef.js:53362:35)
      at Object.transform (C:\..\vben-admin-thin-next\node_modules\vite\dist\node\chunks\dep-6f99a5ef.js:53527:26)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:94:5)
      at async transformRequest (C:\..\vben-admin-thin-next\node_modules\vite\dist\node\chunks\dep-6f99a5ef.js:60120:29)    
      at async C:\..\vben-admin-thin-next\node_modules\vite\dist\node\chunks\dep-6f99a5ef.js:60211:32
```

**Comment by anncwb** [comment]
我在检查下

**Comment by anncwb** [comment]
拉取最新代码在试试

**Comment by high9527** [comment]
thank u!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "anncwb/vben-admin-thin-next"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2021年1月26日(星期二) 凌晨0:07
收件人:&nbsp;"anncwb/vben-admin-thin-next"<vben-admin-thin-next@noreply.github.com&gt;;
抄送:&nbsp;"邓鑫杰"<77753821@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [anncwb/vben-admin-thin-next] 拉到本地后，运行yarn serve 报错。 (#8)





 
Closed #8.

**Comment by high9527** [comment]
勃利县智慧城市建设办公室:邓鑫杰
 
发件人： 邓鑫杰
发送时间： 2021-01-26 08:50
收件人： anncwb/vben-admin-thin-next
主题： 回复： [anncwb/vben-admin-thin-next] 拉到本地后，运行yarn serve 报错。 (#8)
thank u!


------------------ 原始邮件 ------------------
发件人: "anncwb/vben-admin-thin-next" <notifications@github.com>;
发送时间: 2021年1月26日(星期二) 凌晨0:07
收件人: "anncwb/vben-admin-thin-next"<vben-admin-thin-next@noreply.github.com>;
抄送: "邓鑫杰"<77753821@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [anncwb/vben-admin-thin-next] 拉到本地后，运行yarn serve 报错。 (#8)

Closed #8.
